CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 This release adds experimental trace propagation support. When enabled, by setting `propagateTraceContext` to `true` in the `networkRequestCallback` handler, a `traceparent` header will be added to the outgoing network request.
 
+### Changed
+
+- Renamed `@bugsnag/react-navigation-performance` to `@bugsnag/plugin-react-navigation-performance` [#429](https://github.com/bugsnag/bugsnag-js-performance/pull/429)
+- Renamed `@bugsnag/react-native-navigation-performance` to `@bugsnag/plugin-react-native-navigation-performance` [#429](https://github.com/bugsnag/bugsnag-js-performance/pull/429)
+
```

Comment:
@tomlongridge updated, but I'm still not sure this is right... according to [keepachangelog](https://keepachangelog.com/en/1.1.0/) the Deprecated header does seem appropriate here